reporting/src/main/java/org/wso2/testgrid/reporting/model/performance/PerformanceReport.java

```diff
@@ -0,0 +1,98 @@
+/*
+* Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.testgrid.reporting.model.performance;
+
+import org.wso2.testgrid.reporting.ReportingException;
+import org.wso2.testgrid.reporting.renderer.Renderable;
+import org.wso2.testgrid.reporting.renderer.RenderableFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Model class representing the Performance report
```

Comment:
Please end the first line of docs with a dot at the end. Checkstyle in Windows fails otherwise. Let's follow it from next PR onwards.. :)